packages/smooth_app/lib/pages/product/product_image_page.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:photo_view/photo_view.dart';
+
+import 'package:openfoodfacts/model/Product.dart';
+import 'package:openfoodfacts/openfoodfacts.dart';
+
+class ProductImagePage extends StatefulWidget {
```

Comment:
AFAIK it should be a `StatelessWidget`, shouldn't it?

Reply:
Agreed, I changed it to a StatelessWidget.